mitodl/micromasters
# navbar changes

#### What are the relevant tickets?

closes #2564 

#### What's this PR do?

This puts the navbar on all desktop-width pages, and changes around which links show up for staff, which for learners, etc. Check the issue for details on the changes there.

#### How should this be manually tested?

go to any page. you should see a little hamburger in the upper left, which should open the nav menu. there's a little chevron icon which, when clicked, should close the menu again. clicking outside the menu should also close it.

If you're a staff user you should see the following:

![asdfasdfasdf](https://cloud.githubusercontent.com/assets/6207644/23760729/98c95ae2-04be-11e7-9fd4-703f30cb51c0.png)

if you're not you should see:

![fasfffjfj](https://cloud.githubusercontent.com/assets/6207644/23760756/a9782134-04be-11e7-85b9-ef6f214d3aa4.png)

to simulate 'learner' status you can set `SETTINGS.roles` to `[]`.
